liblll/Compiler.h

```diff
@@ -30,9 +30,11 @@ namespace dev
 namespace eth
 {
 
+using ReadCallback = std::function<std::string(std::string const&)>;
```

Comment:
Why defined twice?

Reply:
Because liblll is horribly designed and right now I am not looking to fix that, just fix this simple issue in order to finish #2170.